web/src/components/navbar/NavBarLandingDesktop.tsx

```diff
@@ -18,11 +19,7 @@ export const NavBarLandingDesktop = (): ReactElement => {
           "grid-container-widescreen desktop:padding-x-7 height-8 flex flex-justify flex-align-center flex-wrap"
         }
       >
-        <img
-          className="height-4 margin-y-2"
-          src="/img/Navigator-logo@2x.png"
-          alt="Business.NJ.Gov Navigator"
-        />
+        <img className="height-4 margin-y-2" src="/img/Navigator-logo@2x.png" alt={`${seoTitle.title}`} />
```

Comment:
[sand]
```suggestion
        <img className="height-4 margin-y-2" src="/img/Navigator-logo@2x.png" alt={seoTitle.title} />
```

Reply:
Totally, will fix after lunch (bad habit of mine)